Vostok.Metrics.System/Process/NativeMetricsCollector_Linux.cs

```diff
@@ -132,6 +143,28 @@ private ProcessStatus ReadProcessStatus()
             return result;
         }
 
+        private ProcessCgroupStatus ReadCgroupStatus()
+        {
+            var result = new ProcessCgroupStatus();
+
+            if (File.Exists(cgroupMemoryLimitFileName))
+            {
+                if (long.TryParse(cgroupMemoryLimitReader.ReadFirstLine(), out var memoryLimit))
+                    result.MemoryLimit = memoryLimit;
+            }
+
+            if (File.Exists(cgroupCpuCfsPerioFileName) && File.Exists(cgroupCpuCfsQuotaFileName))
```

Comment:
We can introduce a new method `TryReadFirstLine` in `ReusableFileReader` so that these checks look better.

Our codestyle also does not allow nesting if statements without purpose like in this case. Instead, we merge conditions in one statement.